.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,33 @@
+name: "CodeQL"
+
+on: pull_request
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
```

Comment:
do we need this line?

Reply:
I don't think so, it was in the auto-generated one but I'll remove it

Reply:
As discussed elsewhere we do need this